lib/rubygems/user_interaction.rb

```diff
@@ -679,6 +679,7 @@ def initialize
     begin
       reader = File.open('/dev/null', 'r')
       writer = File.open('/dev/null', 'w')
+    rescue Errno::EINVAL
     rescue Errno::ENOENT
```

Comment:
This change is the same as:

```ruby
rescue Errno:EINVAL
  # Do nothing.
rescue Errno::ENOENT
```

If you want to treat them the same, what you want is:

```ruby
rescue Errno::EINVAL, Errno::ENOENT
```